test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
Ok this make things clearer, but in all these examples the `default_option` in PostSerializer was always ignored/overridden.
Could you please list the same three expected results, but this time assuming that in BlogSerializer there's a `default_include: 'posts'`?